### PR TITLE
MHOUSE-1860: reflect bindAddrs array to bindAddr string in mongohouse

### PR DESCRIPTION
**Pull request by sdorr-mdb**
MHOUSE-1860 (merged today: https://github.com/10gen/mongohouse/pull/1839) changed the bindAddrs array to a simple string.  atlas_data_lake/config.yml needs to reflect that change to allow continued successful testing  :)

### CODE REVIEW

File: .evergreen/atlas_data_lake/config.yml
Comment (jmikola) [inline .evergreen/atlas_data_lake/config.yml:6]: Is `bindAddr` is a new config key introduced by [MHOUSE-1860](https://jira.mongodb.org/browse/MHOUSE-1860)? Is `bindAddrs` being kept in place for BC or removed in favor of this new key?

IIRC, [`build-mongohouse-local.sh`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh) builds the master branch of mongohouse. Do I assume correctly that you'd plan to merge this PR immediately after MHOUSE-1860 is merged to avoid a situation where our config file fails to parse?

The change itself LGTM.

**Review by jmikola — APPROVED** [review]

File: .evergreen/atlas_data_lake/config.yml
Comment (sdorr-mdb) [reply .evergreen/atlas_data_lake/config.yml:6]: > Is `bindAddr` is a new config key introduced by [MHOUSE-1860](https://jira.mongodb.org/browse/MHOUSE-1860)? Is `bindAddrs` being kept in place for BC or removed in favor of this new key?

It's been removed.

> IIRC, [`build-mongohouse-local.sh`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh) builds the master branch of mongohouse. Do I assume correctly that you'd plan to merge this PR immediately after MHOUSE-1860 is merged to avoid a situation where our config file fails to parse?

The mongohouse repo already has this change (was merged yesterday).  And, yes, the order of these commits was intentional so as to avoid parsing issues on your end. :)

> The change itself LGTM.

Thanks!

**Comment by sdorr-mdb** [comment]
evergreen merge

**Pull request merged**
